### PR TITLE
Enable nomos version in OSS (#421)

### PR DESCRIPTION
**Pull request by victorpras**
Enable nomos version in OSS

### CODE REVIEW

File: cmd/nomos/util/write.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ const (
 	NotConfiguredMsg = "NOT CONFIGURED"
 	// UnknownMsg indicates that a field's value is unknown or unavailable.
 	UnknownMsg = "UNKNOWN"
+	// OSSMsg indicates OSS installation
+	OSSMsg = "OSS Installation"
```
Comment (sdowell) [inline cmd/nomos/util/write.go:35]: I'm not sure that `OSS Installation` is the right language here. I think something along the lines of "standalone installation` is more accurate. The important distinction IMO is whether the ACM operator is installed

Comment (sdowell) [reply cmd/nomos/util/write.go:35]: Oh is this a cherry pick? I didn't realize this was already submitted. In that case don't change this PR

Comment (victorpras) [reply cmd/nomos/util/write.go:35]: I can create a separate PR to address this

Comment (nan-yu) [reply cmd/nomos/util/write.go:35]: Good point. @victorpras Can you make a separate PR to change it to `Standalone Config Sync`?

**Review by nan-yu — APPROVED** [review]
/lgtm

File: cmd/nomos/util/write.go (hunk 1 above)
Comment (victorpras) [reply cmd/nomos/util/write.go:35]: PR to change the output https://github.com/GoogleContainerTools/kpt-config-sync/pull/436

**Pull request merged**